### PR TITLE
Post UP release

### PR DESCRIPTION
**Pull request by zertosh**

### CODE REVIEW

**Comment by zertosh** [comment]
@ssorallen, this is missing some flow stuff - but we can chat about it IRL

File: package.json [hunk 1]
```diff
@@ -1,37 +1,34 @@
 {
+  "name": "hyperclick",
+  "version": "0.0.0",
```
Comment (ssorallen) [inline package.json]: Should this be `0.36.0` or something higher than `0.0.35` at least?

Comment (zertosh) [inline package.json]: Meant to leave that at 0.0.35, and do version bump separately

**Comment by ssorallen** [comment]
LGTM 👍

**Comment by zertosh** [comment]
Published as hyperclick@v0.0.36

**Pull request merged**